projects/packages/search/src/widgets/class-search-widget.php

```diff
@@ -182,43 +175,43 @@ public function enqueue_frontend_scripts() {
 			'jetpack-search-widget',
 			plugins_url( 'search/js/search-widget.js', __FILE__ ),
 			array(),
-			JETPACK__VERSION,
+			Helper::get_asset_version( __FILE__ ),
 			true
 		);
 
+		// TODO: No need to enqueue the style for Jetpack the plugin as it's concatenated together.
+		// @see https://github.com/Automattic/jetpack/blob/b3de78dce3d88b0d9b283282a5b04515245c8057/projects/plugins/jetpack/tools/builder/frontend-css.js#L52.
 		wp_enqueue_style(
 			'jetpack-search-widget',
 			plugins_url( 'search/css/search-widget-frontend.css', __FILE__ ),
```

Comment:
Should we update the paths here as well, and remove `search`? I think that part of the path doesn't exist anymore.

Reply:
Thanks for pointing that out @jeherve 
Changed to use `Assets` to enqueue assets.